### DISCUSSION TITLE
Suggestion: Train route overlay

### DISCUSSION THREAD

**Discussion by Davidbtnl** [voxeltycoon/issues#653] [UI&UX] [answered]
Problem: 
Setting up train routes, but constant getting the message that my train can't find a route to the next station when creating more complex rail systems.

Suggestion: 
A train route overlay to see how far the train comes when creating a route. So you can instantly see how far the train can get, and where a signal or connection is missing.

**Comment by katepolak** [comment, marked as answer]
This is a duplicate of #538, please take a look at the suggestions there and feel free to propose your own if you feel like it's missing.
